addons/godottpd/http_server.gd

```diff
@@ -2,6 +2,8 @@
 extends Node
 class_name HttpServer
 
+# If debug messages should be printed in console on requests
+var _debug: bool = false
```

Comment:
We could use print_debug for sure. Eventually we could change the "debug" logic to a "logging" logic (if enabled, both `print_debug` and a `log` function could be called in order to write http info in a log file). I think that this feature would be very important in a prod environment, since with print_debug an exported project won't print anything, and without a logging system users can't debug anything.
What do you think? @dploeger

Reply:
Yes. Maybe we can integrate an already-existing library like [Logger](https://godotengine.org/asset-library/asset/1042) for this. (just searched the asset lib, don't know how good that library is)

For now it would be okay to just use print_debug and implement it in a different PR if you want, so that this doesn't blow up more.

Reply:
I'll go for the Logger. The library you shared looks nice but we could make one even smaller and simplier to only log HTTP Requests/Response (which is generally what some libraries do like [Logbook](https://github.com/zalando/logbook) or [Logback](https://logback.qos.ch/) do).